app/scripts/controllers/transactions/lib/util.js

```diff
@@ -12,7 +12,7 @@ const normalizers = {
   to: (to, lowerCase) =>
     lowerCase ? addHexPrefix(to).toLowerCase() : addHexPrefix(to),
   nonce: addHexPrefix,
-  value: addHexPrefix,
+  value: fixHexValue,
```

Comment:
Nit: could we move this to the `validateTxParams` function instead? It would make more sense semantically if it was there, because we aren't normalizing the value anymore.